lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -405,7 +400,12 @@ def launch_all_nodes
           instance = create_instance(host, ami_spec, nil)
           instances.push({:instance => instance, :host => host})
         end
+        # Add metadata tags to each instance
```

Comment:
@hunner the restriction I was referring to was inability to specify tags during instance creation for v1.

Given this [blog post](https://aws.amazon.com/blogs/developer/upcoming-stable-release-of-aws-sdk-for-ruby-version-2/), it looks like we could drop in v2 of this gem beside the v1 implementation. Then we could change the `create_instance` method to use the v2 implementation, which would allow beaker to send tags in the original `create_instance` method and remove the `add_tags` method entirely.

A quick survey of the code shows that mapping the config hash to the v2 [create_instances](http://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Resource.html#create_instances-instance_method) should be relatively straightforward. I'm partial to fixing this issue in the way I described, as it fixes the broader issue.

Is there a pressing need to get this patch in for a release? You mentioned the f5 ticket, but I'm not clear if there is an impending urgent release for f5 scheduled.